docs/environment/spack.md

```diff
@@ -0,0 +1,456 @@
+
+    -  <a href="https://spack-tutorial.readthedocs.io/en/latest/" target="_blank">**Spack Tutorial**</a>  
+
+    - <a href="https://packages.spack.io/" target="_blank"> **Package Index**  </a> 
+
+    -  <a href="https://spack.readthedocs.io/en/latest/package_fundamentals.html" target="_blank">**Package Fundamentals**</a>   
+
+    - <a href="https://spack.readthedocs.io/en/latest/environments.html" target="_blank">**Spack Environments**</a>  
+
+    - <a href="https://cache.spack.io/" target="_blank"> **Spack Build Cache** </a>  
+
+
+=== "Additional Resources"
+
+    - <a href="https://spack.readthedocs.io/en/latest/configuration.html" target="_blank">**Spack Configuration file **</a>  
+
+    -  <a href="https://spack.readthedocs.io/en/latest/packaging_guide.html" target="_blank">**Advance Packaging Guide**</a> 
+
+    - <a href="https://spack.readthedocs.io/en/latest/build_settings.html" target="_blank">**Concretization Settings**</a>  
+
+
+    - <a href="https://github.com/spack/spack/discussions" target="_blank">**Spack Community & Discussions**</a>  
+
+???+ tabs "Spack Resources"
+
+    === "Important Resources"
+
+        -  <a href="https://spack.readthedocs.io/en/latest/" target="_blank">**Official Documentation**</a>  
+
+        -  <a href="https://spack-tutorial.readthedocs.io/en/latest/" target="_blank">**Spack Tutorial**</a>  
+
+
+        -  <a href="https://spack.readthedocs.io/en/latest/package_fundamentals.html" target="_blank">**Package Fundamentals**</a>  
+
+        - <a href="https://packages.spack.io/" target="_blank"> **Package Index**  </a>  
+
+        - <a href="https://spack.readthedocs.io/en/latest/environments.html" target="_blank">**Spack Environments**</a>  
+
+        
+        - <a href="https://cache.spack.io/" target="_blank"> **Spack Build Cache** </a>  
+
+
+    === "Additional Resources"
+
+        - <a href="https://spack.readthedocs.io/en/latest/configuration.html" target="_blank">**Spack Configuration file **</a>  
+
+        -  <a href="https://spack.readthedocs.io/en/latest/packaging_guide.html" target="_blank">**Advance Packaging Guide**</a>  
+
+        - <a href="https://spack.readthedocs.io/en/latest/build_settings.html" target="_blank">**Concretization Settings**</a>  
+
+        - <a href="https://github.com/spack/spack/discussions" target="_blank">**Spack Community & Discussions**</a>  
+
+
+
+
+
+## Setting up Spack.
+
+
+!!! note
+    The guide is also applicable to other HPC clusters where users need to manage components such as MPI libraries, compilers, and other software through the `module` system.
+
+
+### Connection to a compute node
+For all tests and compilation with Spack, it is essential to run on a **compute node**, not in the login/access node. Here's an example of how to allocate an [interactive session](../jobs/interactive.md) in **Aion cluster**.
+
+```{.sh .copy}
+si -N 1 -n 16 -c 1 -t 0-02:00:00 # on iris: -C broadwell or -C skylake
```

Comment:
Comment on why `-n 16`. You might just want to leave this as `si` after all, the user should be directed to the good job submission docs if they need more.